modules/install/pages/init-setup.adoc

```diff
@@ -250,26 +250,65 @@ Once the [.ui]*Dashboard* screen appears, the new node has been successfully add
 
 Rather than using the Couchbase Web Console, you may elect to initialize your Couchbase cluster by means of the Couchbase Command Line Interface (CLI).
 
-The following CLI syntax can be used for the initial set-up of a single-node Couchbase Server cluster.
-It allows the establishing of administrative credentials, and of port number.
-It adds all services; sets separate RAM quotas for Data, Index, Search, Eventing, and Analytics services, and sets the Index Storage Setting (the default being to support memory-optimized global indexes):
+First you must use the xref:cli:cbcli/couchbase-cli-node-init.adoc[node-init] command to initialize the first node in the cluster.
+The following CLI syntax sets the locations where data files for each service will be stored on the node, sets the location of an alternative JRE on the node, and specifies the hostname that the cluster will use for this node.
 
-[source,bash]
+[source,bash,subs=+quotes]
```

Comment:
Just curious, what does the `subs=+quotes` attribute do?

Reply:
It enables formatting within the code block (in this case, _ _ for replaceable text)